packages/manager/apps/web/client/app/hosting/cdn/order/translations/Messages_fr_FR.json

```diff
@@ -1,29 +1,32 @@
 {
   "hosting_cdn_order_title": "Vous allez souscrire l’option Shared CDN",
   "hosting_cdn_upgrade_title": "Migrer vers l’option Shared CDN",
-
+  "hosting_cdn_order_choose_plan": "Choisissez le plan parmi l'un des plans suivants",
+  "hosting_cdn_order_choose_plan_cdn-advanced": "CDN Advanced",
+  "hosting_cdn_order_choose_plan_cdn-advanced_details": "(Inclus CDN Basic et CDN Security) Optimisez votre configuration CDN selon vos besoins pour garantir une expérience utilisateur sur mesure.",
+  "hosting_cdn_order_choose_plan_cdn-basic": "CDN Basic",
+  "hosting_cdn_order_choose_plan_cdn-basic_details": "En un clic, bénéficiez d'un CDN puissant pour développer facilement votre activité.",
+  "hosting_cdn_order_choose_plan_cdn-basic-free": "CDN Basic",
```

Comment:
Twice CDN Basic ?

Reply:
Yep but they relates to two different plancodes which stand for the same range name.